analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Binary.cs

```diff
@@ -657,4 +657,30 @@ public void Binary_StringConcatenation_Compund_CS(string expression)
         validator.ValidateContainsOperation(OperationKind.Binary);
         validator.ValidateTag("S", x => x.Should().HaveOnlyConstraint(ObjectConstraint.Null)); // FIXME: s is not null here (https://github.com/SonarSource/sonar-dotnet/issues/7111)
     }
+
+    [TestMethod]
+    public void Binary_Add_UpdatesNumberConstraint_CS()
```

Comment:
Not needed, we will most likely not add VB versions
```suggestion
    public void Binary_Add_UpdatesNumberConstraint()
```